src/lib/components/healthselectorv2/HealthSelector.component.test.js

```diff
@@ -0,0 +1,51 @@
+import HealthSelector from './HealthSelector.component';
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('HealthSelector', () => {
+  test('it should display correctly without any props and select first option', () => {
```

Comment:
nit: you can use `it` directly to define a test case, so that you can skip the 'it' in the test case definition
```suggestion
  it('should display correctly without any props and select first option', () => {
```